templates/components/panels/domains.hbs

```diff
@@ -8,7 +8,7 @@
     </header>
 
     <div class="flex-none flex-l flex-row">
-      <p class="flex-grow-1">
+      <p class="flex-grow-1" style="padding-bottom:25px">
```

Comment:
```suggestion
       <p class="flex-grow-1 pb2">
```

`pb2` or `pb3` might perhaps work better. Alternatively the next section could have a `mt2` or `mt3` which is generally more consistent with how styles have been applied throughout the rest of the site.